src/account/azext_account/tests/latest/test_account_scenario.py

```diff
@@ -15,87 +16,79 @@
-    def test_account(self, resource_group):
-
-        self.cmd('az account subscription create '
-                 '--billing-account-name "0aa27f2b-ec7f-5a65-71f6-a5ff0897bd55:ae0dae1e-de9a-41f6-8257-76b055d98372_2019-05-31" '
-                 '--billing-profile-name "27VR-HDWX-BG7-TGB" '
-                 '--cost-center "135366376" '
-                 '--display-name "Contoso MCA subscription" '
-                 '--sku-id "0001" '
-                 '--invoice-section-name "JGF7-NSBG-PJA-TGB"',
-                 checks=[])
-
-        self.cmd('az account subscription create-subscription-in-enrollment-account '
-                 '--display-name "Test Ea Azure Sub" '
-                 '--offer-type "MS-AZR-0017P" '
-                 '--enrollment-account-name "73f8ab6e-cfa0-42be-b886-be6e77c2980c"',
+    # @ResourceGroupPreparer(name_prefix='cli_test_account')
+    def test_account(self):
+        self.kwargs.update({
+            'alias_name': self.create_random_name(prefix='cli_alias', length=24),
+            'new_alias_name': self.create_random_name(prefix='cli_alias_new', length=24),
+            'display_name': "My Subscription",
+            'new_display_name': "My Big Subscription",
+            'billing_scope': "/providers/Microsoft.Billing/billingAccounts/9147924/enrollmentAccounts/253727"
+        })
+
+        self.cmd('az account alias create --name {alias_name} --billing-scope "{billing_scope}" --display-name "{display_name}" --workload "Production"',
+                 checks=[self.check('name', '{alias_name}'),
+                         self.check('properties.provisioningState', 'Succeeded')])
+
+        alias_sub = self.cmd('az account alias show -n {alias_name}',
+                             checks=[self.check('name', '{alias_name}'),
+                                     self.check('properties.provisioningState', 'Succeeded')]).get_output_in_json()
+        sub_id = alias_sub['properties']['subscriptionId']
+        self.kwargs.update({'subscription_id': sub_id})
+
+        # response different from swagger, causing deserialization error
+        # self.cmd('az account alias list',
+        #          checks=[])
+
+        self.cmd('az account subscription list',
+                 checks=[self.greater_than('length(@)', 0)])
+
+        self.cmd('az account subscription show --subscription-id {subscription_id}',
+                 checks=[self.check('displayName', '{display_name}'),
+                         self.check('state', 'Enabled'),
+                         self.check('subscriptionId', sub_id)])
+
+        self.cmd('az account subscription list-location --subscription-id {subscription_id}',
+                 checks=[self.greater_than('length(@)', 0)])
+
+        self.cmd('az account subscription cancel --subscription-id {subscription_id} --yes',
+                 checks=[self.check('subscriptionId', '{subscription_id}')])
+        if self.in_recording:
+            time.sleep(300)
```

Comment:
sleep is mocked. `if self.in_recording` is not necessary.
Why 300? Is it an asynchronous operation?

Reply:
yes. it is asynchronous.